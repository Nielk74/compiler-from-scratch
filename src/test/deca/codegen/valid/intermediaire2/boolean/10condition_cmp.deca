{
int a = 5;
int b = 10;
boolean t = true;
boolean f = false;
if((a < b) && (a <= 7) && (a == 5) && (t == true) && (b > a) && (b >= 10) && (f != t) && (10 != 7)){
    println("(a < b) && (a <= 7) && (a == 5) && (t == true) && (b > a) && (b >= 10) && (f != t) && (10 != 7))");
}else{
    println("!((a < b) && (a <= 7) && (a == 5) && (t == true) && (b > a) && (b >= 10) && (f != t) && (10 != 7)))");
}
}
