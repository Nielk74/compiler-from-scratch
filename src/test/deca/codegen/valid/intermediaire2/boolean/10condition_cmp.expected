(a < b) && (a <= 7) && (a == 5) && (true == true) && (b > a) && (b >= 10) && (false != true) && (10 != 7))
